### ISSUE TITLE
[Keyboard Navigation – BotKit - Docs]: Multiple <h1> headings are defined in “Docs” introduction page.

### ISSUE THREAD

**Issue by v-gasikk** [howdyai/botkit-website#5]
![MAS2 4 6_BotKit_Docs_Headings](https://user-images.githubusercontent.com/48272005/54333144-5d183280-4647-11e9-915b-db8a92e46743.png)
**User Experience:**
Users who rely on keyboards will affect to navigate the page, if Multiple h1 headings are defined.

**Test Environment:**
OS: Win 10 Version 1809 (OS Build 17763.316)
URL: https://botkit.ai/
Browser: Edge
Tool Used: Developer Tool bar (F 12)

**Repro Steps:**
Step 1: Open the above URL in Edge browser.
Step 2: Navigate to “Docs” link and select it.
Step 3: Navigate to “Introduction” link and select it.
Step 4: Navigate to all Headings in the page.
Step 5: Open Developer Tool bar (F 12)
Step 6: Verify whether multiple h1 headings are defined in the page or not.

**Actual Result:**
Multiple h1 headings are defined in “Docs” introduction page.

**Expected Result:**
Single h1 heading should be defined in “Docs” introduction page.

**MAS Reference Link:**
[](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0cf80872-800d-4643-90cf-0c1c3d3e6260})

**Comment by benbrown** [comment]
fixed

**Comment by v-gasikk** [comment]
Issue is Fixed. Hence Closing the bug.
